src/CodeGen_C.cpp

```diff
@@ -2304,12 +2304,12 @@ string CodeGen_C::print_scalarized_expr(const Expr &e) {
     Type t = e.type();
     internal_assert(t.is_vector());
     string v = unique_name('_');
-    stream << get_indent() << print_type(t, AppendSpace) << v << ";\n";
+    stream << get_indent() << print_type(t, AppendSpace) << v << " = " << print_type(t) + "_ops::broadcast(0);\n";
```

Comment:
I wonder if we shouldn't do this, and instead find a way to suppress the warning or just ignore it. It seems like it could hurt performance in some cases?

Reply:
Reverted it back.